pom.xml

```diff
@@ -13,6 +13,13 @@
 	<description>A kafka health check for spring boot actuator</description>
 	<url>https://github.com/deviceinsight/kafka-health-check</url>
 
+	<parent>
```

Comment:
@pvorb
Any reservations against this approach? We use `spring-boot-dependencies` in our internal projects to keep the dependency versions unified across all projects.